### DISCUSSION TITLE
Start Libre 3 sensor with Android 7 phone or on BT only?

### DISCUSSION THREAD

**Discussion by th122** [j-kaltes/Juggluco#129] [Q&A] [answered]
From what I've gathered so far, I'm asking the impossible. My AndroidAPS phone has no NFC, and "the other phone" (Galaxy S6 edge) is limited to Android 7.
Is there any chance to get a Libre3 sensor up and running, so Juggluco could take over?

Where I've looked so far:
- Juggluco
- Libre 3 app
- Patched Libre 3 app

but none of those would be able to run on the Android-7phone.

Thanks in advance for any ideas

**Comment by j-kaltes** [comment, marked as answer]
That's impossible with these phones. For Libre3 you need a phone with NFC running at least Android 8. And even a phone running Android 13 or 14 should be pretty good if you want an uninterrupted connection to the sensor.

**Comment by th122** [comment]
I'm getting there (hopefully...) . Spent from yesterday evening to 4am putting the toolchain together to upgrade the little S6 to Android 11, (Lineage 18.1). 4am saw success, and Juggluco installed without complaints. Now I can read up on Juggluco's documentation how to set up the handover/mirror from one device to the other, and give the thing a test-run. Hooray!

Love the feature set of Juggluco, btw - we had experimented with receiving data from two Dexcom sensors (running two separate instances of xDrip, modified to coexist peacefully), but due to the transition to the G6 didn't pursue that any further.
I had liked the idea of having the sensors overlap, and choose the one to loop on based on their stability (and maybe ask for a third (bloody) opinion when they disagree. With the -now no longer viable- option to run a sensor well past its 10 days, it seemed an obvious improvement towards continuous (and more reliable) data.

My current squeeze (Dexcom running out without notification/warning, and no more spares thanks to Dexcom's new policy of not replacing dead Transmitters that are well within the  "use before" date printed on the box has been resolved by the generous gift from another local looper, so I'll probably wait until that Transmitter runs out to test the Libre3 (given to me on Friday - that one is well past its use-by date (Jan 2023). I'll see how graceful these things are ageing... The perfectly new starter box with Libre-3 handheld device and four sensors that had come my way last Summer I had already passed on to a new involuntary member of the TxD-club, and if the aged sensor comes through, the little updated phone may well go that way as well.
